packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -188,59 +200,69 @@ echo "building layer with package version $VERSION"
 rm -f package.json package-lock.json
 cd ..
 
-echo "step 4/8: Add extension to layer"
+echo "step 4/9: Add extension to layer"
 mkdir -p extensions
 cp ../include/instana-lambda-extension extensions/
 
-echo "step 5/8: creating local zip file with layer contents"
+echo "step 5/9: creating local zip file with layer contents"
 zip -qr $ZIP_PREFIX .
 mv $ZIP_NAME ..
 popd > /dev/null
 
-echo "step 6/8: publishing $ZIP_NAME as AWS Lambda layer $LAYER_NAME to all regions"
-while read -r region; do
-  echo " - publishing to region $region:"
-
-  # See https://docs.aws.amazon.com/cli/latest/reference/lambda/publish-layer-version.html for documentation.
-  lambda_layer_version=$( \
-    AWS_PAGER="" aws --region $region lambda publish-layer-version \
-      --layer-name $LAYER_NAME \
-      --description "Provides Instana tracing and monitoring for AWS Lambdas (@instana/aws-lambda@$VERSION)" \
-      --license-info $LICENSE \
-      --zip-file fileb://$ZIP_NAME \
-      --output json \
-      --compatible-runtimes nodejs8.10 nodejs10.x nodejs12.x nodejs14.x \
-      | jq '.Version' \
-  )
-  echo "   + published version $lambda_layer_version to region $region"
-  if [[ $lambda_layer_version =~ ^[0-9]+$ ]]; then
-    echo "   + setting required permission on Lambda layer $LAYER_NAME / version $lambda_layer_version in region $region"
-    AWS_PAGER="" aws --region $region lambda add-layer-version-permission \
-      --layer-name $LAYER_NAME \
-      --version-number $lambda_layer_version \
-      --statement-id public-permission-all-accounts \
-      --principal \* \
-      --action lambda:GetLayerVersion \
-      --output text
-  else
-    echo "   + WARNING: Lambda layer version $lambda_layer_version does not seem to be numeric, will not set permissions in region $region"
-  fi
+if [[ -z $SKIP_AWS_PUBLISH_LAYER ]]; then
```

Comment:
The file has become big with a lot of conditions and flags. 

I am wondering if we want to add some prompts (not part of this PR)

e.g.
Are these your inputs? local build bla blub
Do you really want to publish "x y z"?
These are the regions to publish to .... ? Confirm

The prompts can be skipped for e.g. Concourse

Reply:
That would be a nice improvement but I agree, it should be out of scope for this PR. There are already some checks in place that should prevent accidentally overwriting our standard layer and image with local changes, but some additional safe guards might not harm.

Reply:
Sweet <3